Library/Homebrew/cask/artifact/abstract_uninstall.rb

```diff
@@ -396,7 +396,11 @@ def uninstall_rmdir(*directories, command: nil, **_)
               command.run!("/bin/rm", args: ["-f", "--", ds_store], sudo: true, print_stderr: false)
             end
 
-            command.run("/bin/rmdir", args: ["--", resolved_path], sudo: true, print_stderr: false)
+            paths = Dir["#{resolved_path}/*"]
+            uninstall_rmdir(*paths, command: command) unless paths.select(&File.method(:file?)).any?
+
+            status = command.run("/bin/rmdir", args: ["--", resolved_path], sudo: true, print_stderr: false)
```

Comment:
Why are we calling `/bin/rmdir` instead of using `FileUtils.rmdir`?

Reply:
It's because we're calling it with `sudo`.

Reply:
> It's because we're calling it with `sudo`.

I did wonder about this in the past… Is there no way of doing `sudo` with the native Ruby methods?

Reply:
Also, do we still need `print_stderr: false` here, since we now have the `unless` clause prior?